receiver/otlpreceiver/testdata/config.yaml

```diff
@@ -2,51 +2,72 @@ receivers:
   # The following entry initializes the default OTLP receiver.
   # The full name of this receiver is `otlp` and can be referenced in pipelines by 'otlp'.
   otlp:
+    protocols:
+      grpc:
+      http:
   # The following entry demonstrates configuring the common receiver settings:
   # - endpoint
   # This configuration is of type 'otlp' and has the name 'customname' with a full name of 'otlp/customname'
   # ('<type>/<name>'. To reference this configuration in a pipeline, use the full name `otlp/customname`.
   otlp/customname:
-    # The receiver will listen on endpoint: "localhost:9090".
-    endpoint: localhost:9090
+    protocols:
+      grpc:
+        # The receiver will listen on endpoint: "localhost:9090".
+        endpoint: localhost:9090
   # The following entry configures all of the keep alive settings. These settings are used to configure the receiver.
   otlp/keepalive:
-    keepalive:
-      server_parameters:
-        max_connection_idle: 11s
-        max_connection_age: 12s
-        max_connection_age_grace: 13s
-        time: 30s
-        timeout: 5s
-      enforcement_policy:
-        min_time: 10s
-        permit_without_stream: true
+    protocols:
+      grpc:
```

Comment:
Is grpc-gateway enabled by "grpc" protocol or by "http" protocol?

Reply:
By http.